### ISSUE TITLE
Confused CK status

### ISSUE THREAD

**Issue by psyhtest** [ctuning/ck#40]
I'm confused about `ck status`:
```
$ ck status
Your version is outdated: V1.6.3
New available version   : V1.6.1

Just execute "ck pull all --kernel" to update CK and all repositories (if you installed CK from GIT) or visit https://github.com/ctuning/ck for more details ...
```

This could be because I installed CK both from PyPi and in the canonical way (by cloning).

**Comment by gfursin** [comment]
It's a bug. Also, I currently check version via Internet via manually changed JSON file on cknowledge.org - I should change it and check version directly from github.com/ctuning/ck ...

**Comment by gfursin** [comment]
Fixed in the latest release (1.6.4). Also now checking version directly from setup.py from GitHub ...
Can you please check and close this ticket...

**Comment by psyhtest** [comment]
No longer getting this message in 1.6.11:
```
anton@localhost / $ ck version
V1.6.11
anton@localhost / $ ck status
Your version is up-to-date: V1.6.11
```
Closing...
